static-site/src/components/ListResources/ResourceGroup.tsx

```diff
@@ -15,9 +15,11 @@ interface ResourceGroupHeaderProps {
   isCollapsed: boolean
   resourcesToShowWhenCollapsed: number
   quickLinks: QuickLink[]
+  tooltipText: string
 }
 
-const ResourceGroupHeader = ({group, isMobile, setCollapsed, collapsible, isCollapsed, resourcesToShowWhenCollapsed, quickLinks}: ResourceGroupHeaderProps) => {
+const ResourceGroupHeader = ({group, isMobile, setCollapsed, collapsible, isCollapsed, resourcesToShowWhenCollapsed, quickLinks, tooltipText}: ResourceGroupHeaderProps) => {
+  if (!tooltipText) tooltipText="Click to load the default (and most recent) analysis";
   const setModalResource = useContext(SetModalResourceContext);
```

Comment:
Suggestion: don't set a default here. Make sure every usage of `ListResources` defines the right text based on the context.